sdk/nodejs/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "${VERSION}",
     "description": "Pulumi's Node.js SDK",
     "license": "Apache-2.0",
-    "repository": "https://github.com/pulumi/pulumi/sdk/nodejs",
+    "repository": "https://github.com/pulumi/pulumi/tree/master/sdk/nodejs",
```

Comment:
It looks like there is actually a documented way to do this "package is nested inside a repo" thing at https://docs.npmjs.com/files/package.json#repository:

----


If the package.json for your package is not in the root directory (for example if it is part of a monorepo), you can specify the directory in which it lives:

```
"repository": {
  "type" : "git",
  "url" : "https://github.com/facebook/react.git",
  "directory": "packages/react-dom"
}
```